### ISSUE TITLE
[Question] Is project dead?

### ISSUE THREAD

**Issue by hitech95** [dadi/cdn#518]
Main website and demo are not working, is this project dead?

**Comment by eshaan7** [comment]
+1. Same question. The docs are up but not the dadi.cloud website.

EDIT: From what I can see, the dadi organization has shut down ? I really like this project, maybe someone could make a fork of this over at https://github.com/jazzband and that could be community maintained.

**Comment by jimlambie** [comment]
Hi @hitech95 @Eshaan7 

The project isn't dead, although its main use is within the Edge network and has been ported to Go. This Node version is still in use in hundreds of projects, I use it frequently. I'm trying to make some time this month to resolve some of the outstanding issues.

Thanks for your interest!

**Comment by eshaan7** [comment]
Thanks for your quick response.

> has been ported to Go

I think maybe that one isn't open source ? I'd very interested in using it since golang would give significantly better performance and concurrency.

**Comment by jimlambie** [comment]
> I think maybe that one isn't open source ? I'd very interested in using it since golang would give significantly better performance and concurrency.

You're right, it isn't open source unfortunately. That being said, I get reasonable performance out of this version as it is, as long as the configuration is tweaked accordingly and the infrastructure it runs on is appropriate. It has different requirements every time!

**Comment by eshaan7** [comment]
I see. Is there any plan on making the go version open source in the future btw?
